modules/renter/contractor/contractor.go

```diff
@@ -147,7 +143,13 @@ func (c *Contractor) Contracts() []modules.RenterContract {
 }
 
 // ContractUtility returns the utility fields for the given contract.
-func (c *Contractor) ContractUtility(id types.FileContractID) (modules.ContractUtility, bool) {
+func (c *Contractor) ContractUtility(pk types.SiaPublicKey) (modules.ContractUtility, bool) {
+	c.mu.Lock()
```

Comment:
any reason you chose the lock instead of rlock?

Reply:
Nope, I can change that.